README.md

```diff
@@ -0,0 +1,56 @@
+# linefeed
+
+A transform stream which converts various line endings into simple newlines, conveniently emitting a single chunk per line.
+
+## Library usage
+
+Convert Windows linefeeds to Unix:
+
+```js
+var fixlines = require('linefeed');
+fs.createReadStream("WINDWS~1.TXT").pipe(fixlines()).pipe(process.stdout);
+```
+
+Or vice versa:
+
+```js
+var LineStream = require('linefeed'),
+    converter = new LineStream({newline:"\r"});
+fs.createReadStream("some_unix_file.txt").pipe(converter).pipe(process.stdout);
+```
+
+Or just grab the lines:
+
+```js
+var LineStream = require('linefeed'),
+    splitter = new LineStream({newline:"", objectMode:true});
+fs.createReadStream("README.md").pipe(splitter).on('data', function (line) {
+    console.log("Got a line:", line);
+});
+```
+
+Besides the usual [stream.Transform](http://nodejs.org/api/stream.html#stream_class_stream_transform) configuration, the only other option is `{newline:string}`. Set it to any desired newline separator, including the empty string (which is especially useful with an object mode transform).
+
+
+## CLI
+
+There is also a minimal shell script you can pipe through if installed globally (or locally, if "./node_modules/.bin/" is in your path):
+
+```sh
+cat README.md | linefeed > README.unix_endings.txt
```

Comment:
I like this example. Using it as a line-ending conversion utility is probably the most realistic cli use. Originally this whole module was created as a supplement for a talk I gave on Node streams, so the CLI only existed to provide a clear correlation to Unix cli philosophy. 

Takeaway... The bin should add a flag to specify a `--newline` character.